src/parsers/tuple_destructuring.cr

```diff
@@ -31,5 +31,9 @@ module Mint
           input: data)
       end
     end
+
+    private def tuple_destructuring_value : Ast::Node?
```

Comment:
I'd call it in the same vein as the other methods, akin to `tuple_destructuring_or_variable`, along with proper `Ast` classes - no casts plz.